client/components/marketing-survey/cancel-jetpack-form/jetpack-cancellation-offer-accepted.tsx

```diff
@@ -0,0 +1,38 @@
+import { useTranslate } from 'i18n-calypso';
+import * as React from 'react';
```

Comment:
Same as above, this could just import the type.
```suggestion
import type { FC } from 'react';
```

Reply:
Nice one!